packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,16 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Uncategorized
+## [30.0.0]
 
+### Added
+
+- Adds a new field `marketData` to the state of `TokenRatesController` ([#4206](https://github.com/MetaMask/core/pull/4206))
+- Adds a new `RatesController` to manage prices for non-EVM blockchains ([#4242](https://github.com/MetaMask/core/pull/4242))
+- Adds a new method to use the multi price endpoint from CryptoCompare ([#4242](https://github.com/MetaMask/core/pull/4242))
```

Comment:
I expected this would line to be sufficient to export it to consumers:
https://github.com/MetaMask/core/blob/85832f4fc1241c741c80725552104b23c203a1d9/packages/assets-controllers/src/crypto-compare-service/index.ts#L1

But I admit I don't know exactly how the exports work

Reply:
Ah, hmm, this is a bit confusing. It looks like this `index.ts` was added to treat the Crypto compare service as a sort of submodule. But it doesn't dictate the exports for the whole module -- that's done by `index.ts` at the root of the package. So if you're ever in doubt, look at that.

Reply:
I see, then it does not appear to be exported and I'll remove from the changelog.